src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -666,6 +666,7 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
         } else if (settings.kind === 'captions' && this._config.enableCEA708Captions) {
           settings.label = settings.label || captionsTextTrackLabels.shift();
           settings.language = settings.language || captionsTextTrackLanguageCodes.shift();
+          settings.emptyLanguage = !!settings.language;
```

Comment:
Maybe instead we can just update the textTracks on the videoElement, like this:
`            if (!!settings.language) {
              settings.language = settings.language;
            } else {
              textTracks[i].language = settings.language = captionsTextTrackLanguageCodes.shift();
            }`
Then we don't need the new prop and it works perfect.

Reply:
you can't update the textTrack on video element it's read only

Reply:
But it worked :)

Reply:
We manipulate the mode, isn't same to manipulate this also?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/TextTrack

Reply:
Yes... You are right. Bummer. Then lets stick with your original solution. Seems to be the best (and only) option